CMSgov/mdct-mcr
# Appease the new linter

# Description


Should unblock `main` by making the linter happy. 

## How to test


No manual testing needed -- pre-commit tests should now be passing.

You can view the pre-commit test by:
1. `cd` into project root
2. if not yet installed globally, run `brew install pre-commit`
3. if not yet installed in this project, run `pre-commit install`
4. run `pre-commit`

## Dependencies

No new deps.

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
